src/bin/server.cpp

```diff
@@ -10,6 +18,10 @@ cxxopts::Options get_server_cli_options() {
     ("help", "Display this help and exit") // NOLINT
     ("address", "Specify the address to run on", cxxopts::value<std::string>()->default_value("0.0.0.0"))  // NOLINT
     ("p,port", "Specify the port number. 0 means randomly select an available one. If no port is specified, the the server will start on PostgreSQL's official port", cxxopts::value<uint16_t>()->default_value("5432"))  // NOLINT
+    ("benchmark_data", "Specify the benchmark name and sizing factor to generate at server start (e.g., " // NOLINT
```

Comment:
Something with "optional"? This could be understood as if this was something normal for the server to do.

Reply:
Changed.